flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "end-of-life": "This application is now being maintained as a Nix flake."
```

Comment:
By the way, this information is not useful for most Flatpak users since most of them do not use NixOS.

Reply:
You don't need to run NixOS to run Nix flakes, they work everywhere.

Reply:
Ah, I did not know that.